### ISSUE TITLE
Tables: Filtering Example - JAWS 2020 cannot navigate non-interactive content in the table

### ISSUE THREAD

**Issue by szet0018** [wet-boew/wet-boew#8935]
JAWS 2020 + Chrome Version 85.0.4183.102
- JAWS can navigate to and read the links in the table, but does not read any of the non-interactive content i.e. the regular text inside the <td>

NVDA + Chrome Version 85.0.4183.102
- NVDA is able to navigate to and read both the links and the non-interactive text in the table

The issue with JAWS not reading the non-interactive content in the table is solved when role="grid" is removed from the table.

From my understanding, role="grid" is added to the table element when the table has interactive content that should be organized in a grid.

However, it seems that role="grid" triggers application mode in some screen readers. This means that they only read the interactive elements, ignoring elements like paragraphs, lists, etc. which are not interactive.

I looked at the WAI-ARIA example of role="grid" on table elements at:
https://www.w3.org/TR/wai-aria-practices-1.1/examples/grid/dataGrids.html

Their solution is to give tables with role="grid" the keyboard interaction pattern of widgets, that is, the user TABs to the widget, then navigates using the ARROW keys. In the WAI-ARIA example the user can use the ARROW keys to navigate all the cells in the table, not just the cells with links.

**Comment by thekodester** [comment]
Just an FYI, but `role="gird"` is added by the plugin https://datatables.net.
